src/services/wallets/JsonWalletConnector.ts

```diff
@@ -1,15 +1,23 @@
+import Arweave from 'arweave';
+import Ar from 'arweave/node/ar';
 import { JWKInterface } from 'arweave/node/lib/wallet';
 import { isString } from 'lodash';
 
 import { ArweaveWalletConnector } from '../../types';
+import { deployedAntQuery } from '../../utils/constants';
+import { arweave } from '../arweave/arweave';
 
 // A lot to do here, r.e. security, we will likely move to a different approach.
 export class JsonWalletConnector implements ArweaveWalletConnector {
   private _walletFile;
   private _wallet?: JWKInterface;
+  private _arweave: Arweave;
+  private _ar: Ar = new Ar();
+  private _address?: string;
```

Comment:
are these needed? they don't seem to be used anywhere

Reply:
_ar is used for parsing balance to winston and address is used in the getAnts query - they are needed.

Reply:
they aren't _yet_ used in this class though 😉

Reply:
ohhhh i didnt read the file name, youre right they can be removed. i think.